win32/win_opendir.c

```diff
@@ -1,336 +1,425 @@
-//
-// 03/10/2006 James Haley
-//
-// For this module only: 
-// This code is public domain. No change sufficient enough to constitute a
-// significant or original work has been made, and thus it remains as such.
-//
-//
-// DESCRIPTION:
-//
-// Implementation of POSIX opendir for Visual C++.
-// Derived from the MinGW C Library Extensions Source (released to the
-// public domain). As with other Win32 modules, don't include most DOOM
-// headers into this or conflicts will occur.
-//
-// Original Header:
-//
-// * dirent.c
-// * This file has no copyright assigned and is placed in the Public Domain.
-// * This file is a part of the mingw-runtime package.
-// * No warranty is given; refer to the file DISCLAIMER within the package.
-// *
-// * Derived from DIRLIB.C by Matt J. Weinstein 
-// * This note appears in the DIRLIB.H
-// * DIRLIB.H by M. J. Weinstein   Released to public domain 1-Jan-89
-// *
-// * Updated by Jeremy Bettis <jeremy@hksys.com>
-// * Significantly revised and rewinddir, seekdir and telldir added by Colin
-// * Peters <colin@fu.is.saga-u.ac.jp>
-//
-
-#ifndef _MSC_VER
-#error i_opndir.c is for Microsoft Visual C++ only
-#endif
+/*
+MIT License
+Copyright (c) 2019 win32ports
```

Comment:
Could you adjust the copyright information in `README.md`, please?

Reply:
Hmm, where do I update it? Chocolate Doom doesn't seem to have a “Legalese” section.

Reply:
Oops, I didn't realize we're not in the woof repo. Sorry, it's still early...